portal/src/main/scala/cromwell/pipeline/datastorage/dto/user/UpdateRequest.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.pipeline.datastorage.dto.user
+
+import play.api.libs.json.{ Json, OFormat }
+
+final case class UpdateRequest(email: String, firstName: String, lastName: String)
```

Comment:
It's better to rename it to `UserUpdateRequest`

Reply:
done